### ISSUE TITLE
Spam to Trash: Make the feature customizable

### ISSUE THREAD

**Issue by takayukister** [takayukister/flamingo#8]
Related issue: #2

Some users may not want to trash items automatically, or may want to change the days from the default 30.

Let's define the constant like the following so that users can override the value in their wp-config.php file.

```
if ( ! defined( 'FLAMINGO_MOVE_TRASH_DAYS' ) ) {
	define( 'FLAMINGO_MOVE_TRASH_DAYS', 30 );
}
```

Also, let's check the FLAMINGO_MOVE_TRASH_DAYS constant in flamingo_schedule_move_trash(), and if the value is 0, abort the trashing process.

**Comment by itpixelz** [comment]
Awesome, I like the idea and updating my code

**Comment by itpixelz** [comment]
I have made FLAMINGO_MOVE_TRASH_DAYS customizable by the user and checking 'if defined' in my code. So now it will override if it is already defined in wp-config.php file by the user. 

If the user sets FLAMINGO_MOVE_TRASH_DAYS value to 0 or in minus, our function flamingo_schedule_move_trash() will abort the process of moving spam to trash.

I have updated line spacing for better readability too.

The update is part of PR #7
